src/structures/vroom/input/input.cpp

```diff
@@ -678,7 +678,7 @@ void Input::set_matrices(unsigned nb_thread) {
           // Durations matrix not manually set so defined as empty
           // above.
           if (_locations.size() == 1) {
-            d_m->second = Matrix<Cost>({{0}});
+            d_m->second = Matrix<Cost>(1);
```

Comment:
Here the [matrix ctor](https://github.com/VROOM-Project/vroom/blob/39438be9e5b61d88a5608a71b369994fc212f49a/src/structures/generic/matrix.cpp#L14-L16) will call `data.resize(1)` so that the underlying storage vector will have the right size. But I think we still need to initialize the single entry with a zero value.

Reply:
`Cost` is an alias of `uint32_t` so there is no ctor to call. However, according to [reference](https://en.cppreference.com/w/cpp/container/vector/resize) in this overload of `std::vector::resize`, the items are default-inserted, which in this case boils down to zero-initialization.

There is no need to assign zero value explicitly, but I can add it for clarity. Please let me know what you think.

Reply:
Thanks for the pointer, I didn't know that. I guess this is fine as is, thanks!